modules/src/ics04_channel/error.rs

```diff
@@ -3,8 +3,13 @@ use thiserror::Error;
 
 pub type Error = anomaly::Error<Kind>;
 
+use crate::ics24_host::identifier::{ConnectionId,ChannelId};
+
 #[derive(Clone, Debug, Error)]
 pub enum Kind {
+    #[error("not implemented yet")]
+    Niy,
```

Comment:
In general, instead of returning a `Niy` error when some function is not implemented yet, you can use the `todo!` macro, eg.

```rust
fn my_unimplemented_function(x: u32) {
  todo!()
}
```

I would thus suggest removing this variant from the `Kind` struct.

Reply:
Thanks. It is a better one also I used it only as an intermediate step and forgot to remove it.